src/backgroundfilter/BackgroundFilterProcessor.ts

```diff
@@ -398,7 +398,7 @@ export default abstract class BackgroundFilterProcessor {
     this.delegate.removeObserver(this.cpuMonitor);
     this.canvasVideoFrameBuffer.destroy();
     this.worker?.postMessage({ msg: 'destroy' });
-    this.worker?.postMessage({ msg: 'close' });
+    this.worker?.postMessage({ msg: 'stop' });
```

Comment:
For learning: Should the `stop` be before `destroy`?

Reply:
Good question. `destroy` should go first because `destroy` is meant to release the internal state of the blur process within the worker whereas `stop` actually ends the worker by calling `self.close() (https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope/close). Ideally, the command should've been named `close` but the API contract is already made, so changing it now isn't worth the effort.

Released worker code for reference https://static.sdkassets.chime.aws/bgblur/workers/worker.js. I can also ping you the internal package if you'd prefer to look at that instead.

Reply:
Ok makes sense, thanks for the explanation :)